osm_validator_front/js/index.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import axios from 'axios'
+import { Provider } from 'react-redux'
+import { HashRouter, hashHistory } from 'react-router-dom'
+import configureStore from './store/configureStore.jsx'
+import App from './containers/app.jsx'
+import './../css/main.css';
+import Login from './components/login.jsx';
+
+axios
+    .get('/user/info')
+    .then(function(result) {
+        const store = configureStore({user: result.data.osm_user});
+
+        ReactDOM.render(
+           <Provider store={store}>
+               <HashRouter history={hashHistory}>
+                   <App />
+               </HashRouter>
+            </Provider>,
+            document.getElementById('app')
+        );
+    })
+    .catch(function(error) {
+        ReactDOM.render(
+             <Login />,
+             document.getElementById('app')
+        );
+    })
```

Comment:
We should show some pages for loggined and unloggined users, so better at first load app, and then make requests for state, let's change it.